File: client/src/Component/Pages/Game/Dtos/Game.ts
export interface GameFormDto{
    localteamid: number;
    visitorteamid: number;
    champeonshipid: number;
    amountGoalsLocal?: number;
    amountGoalsVisitor?: number;
    date: string;
}

export interface GamesRandomFormDto{
    localteamid: number;
    visitorteamid: number;
    champeonshipid: number;
    amountGoalsLocal?: number;
    amountGoalsVisitor?: number;
    date: string;
}

File: server/Dtos/Game.cs


using System.ComponentModel.DataAnnotations;

namespace server.Dtos;

public partial class GameDto
{
    [Required]
    public int? Localteamid { get; set; }
    [Required]

    public int? Visitorteamid { get; set; }
    [Required]

    public int? Champeonshipid { get; set; }

    public int? AmountGoalsLocal { get; set; } = null;
    public int? AmountGoalsVisitor { get; set; } = null;
    [Required]
    public DateOnly Date { get; set; }

}


public partial class GamePutDto
{
    [Required]
    public int? Champeonshipid { get; set; }

    public int? AmountGoalsLocal { get; set; } = null;
    public int? AmountGoalsVisitor { get; set; } = null;
    [Required]
    public DateOnly Date { get; set; }

}


public partial class GamesRandomsDto
{
    public int? Champeonshipid { get; set; }
    public int? Divisionid { get; set; }


}

